book.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
     title: 'ZeroMQ RFCs',
-    gitbook: '3.0.0-pre.15'
+    gitbook: '3.0.0-pre.15',
```

Comment:
We release GitBook 3.0.0 as stable (not yet announced because we still want to investigate potential issues), so yes it'll work.